lz4/lz4hc.c

```diff
@@ -313,6 +313,7 @@ static int LZ4HC_compress_generic (
     int const inputSize,
     int const maxOutputSize,
     int compressionLevel,
+    int constLevel,
```

Comment:
The changes in lz4hc.c are not a good idea because:
1. The changes will be removed in the future with a newer version lz4hc.c
2. I change source code of compressors only to fix some errors.

If you have some improvements for a compressor they should be commited directly to this compressor's repo. In this case it's https://github.com/lz4/lz4.